app/models/auth_source.rb

```diff
@@ -17,12 +17,18 @@
 
 class AuthSource < ActiveRecord::Base
   include Authorizable
+
+  attr_accessible :name, :host, :tls, :port, :server_type, :account,
+      :base_dn, :groups_base, :ldap_filter, :onthefly_register,
+      :attr_login, :attr_firstname, :attr_lastname, :attr_mail, :attr_photo
+
   audited :allow_mass_assignment => true
 
   validates_lengths_from_database :except => [:name, :account_password, :host, :attr_login, :attr_firstname, :attr_lastname, :attr_mail]
   before_destroy EnsureNotUsedBy.new(:users)
   has_many :users
   has_many :external_usergroups, :dependent => :destroy
+  attr_protected :type
```

Comment:
attr_protected is pointless. all attributes are blacklisted by default, unless they're listed in attr_accessible, you can remove this line and any others